sous_chef/tests/integration_tests/messaging/test_gmail_api_integration.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+import pytest
+from hydra import compose, initialize
+from sous_chef.messaging.gmail_api import GmailHelper
+
+
+@pytest.fixture
+def gmail_helper():
+    with initialize(config_path="../../../config/messaging"):
+        config = compose(config_name="gmail_api")
+        return GmailHelper(config.gmail)
+
+
+DEFAULT_SUBJECT = "[sous_chef_tests]"
+
+
+class TestGmailHelper:
```

Comment:
Could have it in the future that the script is able to check by itself that the mail arrived, but that might be overkill. I had implemented this for the trading bot, so if ever needed, could refresh some code for it.

Reply:
Sounds like a good idea. I would agree that it's not so critical atm.